docs/application/dotnet/guides/app-management/component-port.md

```diff
@@ -0,0 +1,469 @@
+    To handle the received request, define and the register an event handler for the `RequestReceived` event of the `Tizen.Applications.ComponentBased.ComponentPort` class.
+
+    ```
+    {
+        _port = new ComponentPort("ServerService");
+        _port.RequestReceived += OnRequestReceived;
+        _port.AddPrivilege("http://tizen.org/privilege/datasharing");
+        _thread = new Thread(new ThreadStart(OnThread));
+        _thread.Start();
+    }
+
+    private void OnThread()
+    {
+        Log.Info("OnTrhead()");
+        _port.WaitForEvent();
+    }
+
+    private void OnRequestedReceived(object sender, RequestEventArgs args)
+    {
+        Log.Info("Sender: + args.Sender);
+        var request = args.Request;
+        if (request.GetType() == typeof(Request))
+        {
+            var req = (Request)request;
+            Log.Info("Command: " + req.Command);
+            Log.Info("Sequence: " + req.Sequence);
+            Log.Info("Message: " + req.Message);
+
+            if (args.IsReplyRequested)
+            {
+                args.Reply = new Response(req.Command, req.Sequence, 0);
+            }
+        }
+    }
+    ```
+
+3.  In the sending component, to send the request, follow the steps:
+
+    a. Call the `WaitForEvent()` method `Tizen.Applications.ComponentBased.ComponentPort` class in the thread if you want to receive the reply from the ServerService.
+
+    b. Call the `WaitForPort`() method `Tizen.Applications.ComponentBased.ComponentPort` class to wait until the target port is ready.
+
+    c. Use `Send()` method `Tizen.Applications.ComponentBased.ComponentPort` class to send the request.
+
+    d. Provide the request to be sent as an instance of the `ClientService.Tizen.Request` class as shown in the below code:
+
+    ```
+    {
+        _port = new ComponentPort("ClientService");
+        _port.RequestReceived += OnRequestReceived;
+        _thread = new Thread(new ThreadStart(OnThread));
+        _thread.Start();
+        ...
+        await ComponentPort.WaitForPort("ServerService");
+
+        try
+        {
+            var request = new Request("TestCommand", _seq++, "TestMessage");
+            _port.Send("ServerService", 5000, request);
+        }
+        catch
+        {
+            Log.Error("Failed to send request");
+        }
+        ...
+    }
+
+    private void OnThread()
+    {
+        Log.Info("OnTrhead()");
+        _port.WaitForEvent();
+    }
+
+    private void OnRequestedReceived(object sender, RequestEventArgs args)
+    {
+        Log.Info("Sender: + args.Sender);
+        var response = args.Request;
+        if (response.GetType() == typeof(Response))
+        {
+            var req = (Response)request;
+            Log.Info("Command: " + req.Command);
+            Log.Info("Sequence: " + req.Sequence);
+            Log.Info("Result: " + req.Result);
+        }
+    }
+    ```
+
+<a name="task"></a>
+## Using a Component Task
```

Comment:
Global:

Component Port
Component Task 

These are to be written as is, am I correct? then we need to maintain consistency all over this page. 

```suggestion 
## Use Component Task

Reply:
Done